examples/snippets/modules/ROOT/examples/akkastreams-scala/src/main/scala/cloudflow/akkastreamsdoc/DataHttpIngress.scala

```diff
@@ -0,0 +1,18 @@
+package cloudflow.akkastreamsdoc
+
+// tag::httpIngress[]
+import cloudflow.akkastream._
+import cloudflow.akkastream.util.scaladsl._
+
+import cloudflow.streamlets.avro._
+import cloudflow.streamlets._
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import JsonSupport._
+
+class DataHttpIngress extends AkkaServerStreamlet {
+  val out         = AvroOutlet[Data]("out").withPartitioner(RoundRobinPartitioner)
```

Comment:
very minor, you can leave the `.withPartitioner(RoundRobinPartitioner)` out cause it is default. (maybe for didactic purposes it's better to be explicit so not sure)